Update rule from Redfish v1.20.1 (#126)
cybozu-go/setup-hw: docs/how_to_install_collector.md

How to install collector command
=====================================

> The collector command can run in setup-hw container and bare metal linux server. 
> in this memo describe how to install bare metal server.

Reviewer: ```suggestion
The collector command can run in a setup-hw container or on a bare metal Linux server. 
This memo describes how to install the command on a bare metal server.
```

## 1.Clone code from github repository

```
$ git clone https://github.com/cybozu-go/setup-hw
```

## 2.Build & install collector command

```
$ cd setup-hw
$ make install
```

## 3.Setup config

> Put a bmc-user.json file in /etc/neco/ that must have "support" user due to use the collector command.

Reviewer: ```suggestion
Put a bmc-user.json file in /etc/neco/ that must have "support" user to use the collector command.
```

```
{
  "support": {
    "password": {

>       "raw": "cybozu"

Reviewer: I don't think it's a good idea to put our company name as a password example here, even though this is not a real password.
There are 2 reasons.

1. Some people may misunderstand they should place exactly `"cybozu"` here. We should clarify that the string is just for example.
2. This is a document for OSS users, so it would be good to use organization-neutral examples.

```suggestion
      "raw": "raw password here"
```

    }
  }
}
```

Put a bmc-address.json file in /etc/neco. 

```

> {"ipv4":{"address":"10.210.152.197","netmask":"255.255.252.0","maskbits":22,"gateway":"10.210.152.198"}}

Reviewer: ditto

`192.0.2.0/24` is a reserved IP address block for documentation and examples.

```suggestion
{
  "ipv4": {
    "address": "192.0.2.3",
    "netmask": "255.255.255.0",
    "gateway": "192.0.2.1"
  }
}
```

```

please see [config.md]("config.md") file.

## 4.Check 

By following the above steps, you can execute the collector command.

```
$ collector show
```

> Next, see [how to generate rules](how_to_install_collector.md).

Reviewer: ```suggestion
Next, see [how to generate rules](how_to_generate_rules.md).
```
